File: Back/ApiTcg/Domain/Entities/Set.cs
namespace Domain.Entities;

public class Set
{
    public string Id { get; set; } = "";
    public string Name { get; set; } = "";
    public string? Logo { get; set; }
    public string? Symbol { get; set; }
    public int CardCountTotal { get; set; }
    public int CardCountOfficial { get; set; }

    public ICollection<Card> Cards { get; set; } = [];
}

File: Front/poke-tcg/src/app/models/set/set.ts
import { CardSummary } from "../card/cardSummary";

export interface Set 
{
   id : string;
   name : string;
   logo : string;
   symbol  : string;
   cardCountTotal : number;
   cardCountOfficial : number;
   cards: CardSummary[];
   isCompleted: boolean; 
}
